presto-orc/src/main/java/com/facebook/presto/orc/Stripe.java

```diff
@@ -29,13 +30,15 @@
     private final Map<Integer, ColumnEncoding> columnEncodings;
     private final List<RowGroup> rowGroups;
     private final InputStreamSources dictionaryStreamSources;
+    private final LongDictionaryProvider longDictionaryProvider;
```

Comment:
I think the naming of DictionaryProvider is good, DictionarySource is also reasonable. I personally think the former sounds better, but it would be a relatively small change if presto folks prefer the latter. I'll leave the call to them.